### DISCUSSION TITLE
Feature Request: Show repo visibility in main page

### DISCUSSION THREAD

**Discussion by ArieHein** [community/community#45916] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When you enter the main page, after you login, it shows the Top Repos on the left side, and allows you to show more of them when you have quite a few. I would like to be able to see which of them are set to visibility public or visibility private directly in the main page.

No need to show the words "public" or "private", perhaps some dot with color or other icon or allow it to be configured in the account settings. (of course make sure accessibility readers can also behave correctly).

I have a few repos, and I tend to forget sometimes which ones are public and which are not, a quick visual cue would be great. It might be ok to just show the public one and not show anything near those that are private, or let that also be a configured in the settings.

**Comment by SilasRodrigues19** [comment]
Hey @ArieHein 

It already exists, there is no color to differentiate but there is the option.

![image](https://user-images.githubusercontent.com/47532703/215843390-8c166561-5952-4924-b3e5-82031b1d5163.png)

**Comment by ArieHein** [comment]
Sorry but it doesn't.
If you hover on one you see it, i have 20. Do you think i should hover over 20 just to see if its public or not ?

**Reply by SilasRodrigues19** [reply]
The profile already has a way to see which are public and which are private, it even has the filter

**Reply by ArieHein** [reply]
Sorry, I might have no explained myself well.

When you open the main window. What you see is a list of repositories.
Just by looking at them , i would like a visual indication which are public or not.
Can you point me to the settings that allow this ?

**Reply by SilasRodrigues19** [reply]
It doesn't exist on the main page, but it does exist on the repos page

![image](https://user-images.githubusercontent.com/47532703/215846701-e6f0249c-4fa4-4e4f-869e-05df3077caf3.png)

**Reply by ArieHein** [reply]
Indeed, Which is why I added this as a feature request.
A small visual indicator would be great to add to give that initial visibility for the owner.
